ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/RecoveryTest.java

```diff
@@ -498,6 +505,7 @@ private void shouldRecover(final List<QueuedCommand> commands) {
 
   @Test
   public void shouldRecoverCreates() {
+    topicClient.preconditionTopicExists("test2", 3, (short) 1, Collections.emptyMap());
```

Comment:
why do we need these to exist? 

If the user stopped KSQL, deleted sink topics, and then restarted, wouldn't we expect KSQL to recreate the sink topics and carry on?